neuralcompression/functional/_soft_round/_soft_round_inverse.py

```diff
@@ -0,0 +1,19 @@
+import torch
+
+
+def soft_round_inverse(
+    y: torch.Tensor, alpha: float, eps: float = 1e-3
+) -> torch.Tensor:
+    alpha = torch.tensor(alpha)
+
+    eps = torch.tensor(eps)
+
+    maximum = torch.maximum(alpha, eps)
+
+    m = torch.floor(y) + 0.5
+
+    x = m + torch.clamp(
+        torch.atanh((y - m) * (torch.tanh(maximum / 2.0) * 2.0)) / maximum, -0.5, 0.5
+    )
```

Comment:
nit: could eliminate white space here for readability. One suggestion would be to group the first two calls to `torch.tensor` together, then have the three math operations, and have the final group be the return statement.

Reply:
I don't mind nits! I messed with the formatting. Let me know if it's clearer.